lib/App/Mi6/Release/CheckAuth.rakumod

```diff
@@ -4,7 +4,7 @@ use App::Mi6::JSON;
 use App::Mi6::Util;
 
 method run(*%opt) {
-    my $expect-auth = %opt<expect-auth>;
+    my $expect-auth = %opt<expect-auth>.any;
```

Comment:
This is a nitpick,
but if you use a junction here, can you update the error message?
```
die "auth '$meta-auth' in META6.json does not match auth '$expect-auth' in ~/.fez-config.json";
```
A junction in double quote string generates `any(...)` string; it does not look nice.

Reply:
Can't promise doing it soon. So far, replacing with `%opt<expect-auth>` would already produce better outcome because for a `List` it would result in something like _'user group1 group2'_, including the single quotes. Slightly better message would require a little more time.